### PR TITLE
Fix Missing Config in Test Gen Workflow

### PR DESCRIPTION
**Pull request by EricLeFort**
# Description

Adds the missing user and email to resolve the failing git commit step

# Tests

N/A

**Instruction and/or command lines to reproduce your tests:** N/A

**List links for your tests (use go/shortn-gen for any internal link):** N/A

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [ X ] I have performed a self-review of my code.
- [ X ] I have necessary comments in my code, particularly in hard-to-understand areas.
- [ X ] I have run one-shot tests and provided workload links above if applicable. 
- [ X ] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: .github/workflows/generate.yml [hunk 1]
```diff
@@ -34,6 +34,8 @@ jobs:
     
     - name: Git Commit/Push Changes
       run: |
+        git config --global user.name "Eric Le Fort"
```
Comment (will-cromar) [inline .github/workflows/generate.yml:37]: Can you make this the name and email of the triggering commit?

Comment (EricLeFort) [reply .github/workflows/generate.yml:37]: On it!

Comment (will-cromar) [reply .github/workflows/generate.yml:37]: A placeholder also works. It looks like the current flow uses the github username and a placeholder email, e.g. https://github.com/GoogleCloudPlatform/ml-testing-accelerators/commit/add1bc244580a53b640c9a27c851a332b2495992

File: .github/workflows/generate.yml [hunk 2]
```diff
@@ -34,6 +34,8 @@ jobs:
     
     - name: Git Commit/Push Changes
       run: |
+        git config --global user.name "`git log -1 --pretty=format:'%an'`"
```
Comment (will-cromar) [inline .github/workflows/generate.yml:37]: nit: can you add a comment explaining what these git log commands are doing?

**Review by will-cromar — APPROVED** [review]
Otherwise lgtm

File: .github/workflows/generate.yml (hunk 2 above)
Comment (EricLeFort) [reply .github/workflows/generate.yml:38]: Added

**Pull request merged**